common/docsV1.md

```diff
@@ -1094,7 +1094,7 @@ Details on Provenance Resources are given in the <a href="https://www.hl7.org/fh
 - **Timestamp:** Time when attestation was made.
 - **Reason:** Reason for the attestation (currently only: http://hl7.org/fhir/v3/ActReason#TREAT is supported).
 - **Organization ID:** The agent making the attestation referenced by their Organization Resource ID. 
-  - _Your Organization ID can be found by referencing the {id} variable in the resource object of your practitioner._
+  - _Your Organization ID can be found on the <a href="https://sandbox.dpc.cms.gov/api/v1/Organization">Organization</a> page in the DPC Portal._
```

Comment:
Oops, I meant to the instructions higher in the page

Reply:
oops, my bad